puppet/zulip/files/nginx/zulip-include-frontend/upstreams

```diff
@@ -11,3 +11,7 @@ upstream localhost_sso {
 upstream camo {
     server 127.0.0.1:9292;
 }
+
+upstream tusd {
+    server 127.0.0.1:9900;
+}
```

Comment:
How did you pick this port number?

Reply:
Randomly.  tusd doesn't have a default port.